### PR TITLE
Remove hierarchical conflict docs

### PR DESCRIPTION
**Pull request by calebmer**
Better?

### CODE REVIEW

**Comment by svmnotn** [comment]
[That](https://github.com/calebmer/ardite/pull/1/files#diff-45946bc729f66eed6e2f926036d03b46R47) should parse into something like:

``` json
{
  "profile": {
    "$value": "Budd",
    "about": "I‘m a web developer!"
  }
}
```

Where `$value` is used to represent the value of `/profile`

**Comment by calebmer** [comment]
I agree, I like that syntax better too. However, it seems to me more of an implementation detail (boolean saying nest folder value) instead of the idiomatic behavior.

Its more logical to see:

``` json
{
  "profile": "Budd",
  "profile": {
    "about": "I‘m a web developer!"
  }
}
```

Even if it is a horrible anti-pattern.

I do agree that realistically we should never output JSON which looks like that.

**Comment by calebmer** [comment]
Another consideration is holding that object in memory…

**Comment by svmnotn** [comment]
Holding in memory? Also it is horrible JSON that should never be outputted thus why we should not be mis-documenting it to say such a thing

**Comment by svmnotn** [comment]
However you could add a line saying that `$value` is more of an impl detail that might change so that no one depends on that, although I like it being named value so it might stick

**Comment by calebmer** [comment]
I‘ll definitely add that.

**Comment by calebmer** [comment]
Ok, it has been added.

File: docs/design/02-pointers.md [hunk 1]
```diff
@@ -73,6 +73,17 @@ Like this?
 }
 ```
 
-As you can see, none of these interpretations are accurate. When turning into JSON, Ardite will prefer the second format if the `/profile/about` format is the most recent (recency will be better explained when we talk about the ledger). However, if `/profile` is the most recent, Ardite will prefer the first JSON format. Ardite would never create a document which looks like the third format (for more information on how this is interpreted, see the reducing section of the design docs).
+As you can see, none of these interpretations are very helpful. When turning into JSON the third format is what would naturally be assumed, where the second value will be whatever is the most recent (recency will be better explained when we talk about the ledger). This is a very ugly solution, therefore in most scenarios Ardite prefers the following format:
```
Comment (svmnotn) [inline docs/design/02-pointers.md]: Change to:
As you can see, the first and second one hide parts of the state whilst the third is not valid JSON. Thus Ardite will provide the following format unless a schema is applied:

**Pull request merged**